<launch>
    <node type="virtual_box_counter" pkg="perception" name="virtual_box_counter" output="screen">
        <!-- boundaries of the virtual box -->
        <param name="x_lower_bound" value="0" />
        <param name="x_upper_bound" value="10" />
        
        <param name="y_lower_bound" value="-5" />
        <param name="y_upper_bound" value="5" />
        
        <param name="z_lower_bound" value="0" />
        <param name="z_upper_bound" value="1" />

        <!-- threshold for virtual box  -->
        <param name="points_in_box_threshold" value="50" />

    </node>
</launch>